public/scripts/views/portfolioView.js

```diff
@@ -9,7 +9,7 @@
       $('#content').removeClass('main-intro-big').addClass('main-intro');
```

Comment:
Right now, how you have it set up in your master, you have 3 view files. All three of these files are controllers, and they belong inside of your controller folder. A lot of the data that you have in portfolioView.init can actually be moved to its own functions that you would save in a folder inside of your view repo and then call those functions inside of your init. Your slideshow.js can also be moved into your view folder